business-central-parent/business-central-distribution-wars/business-central/scripts/set-rhdm-props.cli

```diff
@@ -0,0 +1,4 @@
+/system-property=org.kie.workbench.profile:add(value=FORCE_PLANNER_AND_RULES)
+/system-property=org.kie.workbench.profile:read-resource
```

Comment:
Reading resources is not needed here. Please remove it.

Reply:
Yeah you're right, I put them in for debugging.. should remove, thanks for looking ;-)